src/ConnectionAutomaton.h

```diff
@@ -186,7 +186,7 @@ class ConnectionAutomaton final
     return streamsFactory_;
   }
 
-  FrameSerializer& frameSerializer() const override;
+  FrameSerializer& frameSerializer() const ;//override;
```

Comment:
I believe we can remove the method now as the intent for this PR is to not use Frames and FrameSerializer outside of ConnectionAutomaton.